src/Symfony/Bridge/Doctrine/Form/ChoiceList/DoctrineChoiceLoader.php

```diff
@@ -91,9 +91,18 @@ protected function doLoadChoicesForValues(array $values, ?callable $value): arra
             trigger_deprecation('symfony/doctrine-bridge', '5.1', 'Not defining explicitly the IdReader as value callback when query can be optimized is deprecated. Don\'t pass the IdReader to "%s" or define the "choice_value" option instead.', __CLASS__);
         }
 
+        $idReader = null;
+        if (\is_array($value) && $value[0] instanceof IdReader) {
+            $idReader = $value[0];
```

Comment:
whats the purpose of persisting the id reader in `$idReader`? It's not used at atll afterwards except in an if boolean condition.

Reply:
Right, I missed a spot. Fixed.